packages/domain/src/use-cases/publications/CreatePost.ts

```diff
@@ -61,6 +69,27 @@ export type BasePostRequest = {
    * These are not the same as #hashtag in the publication content. Use these if you don't want to clutter the publication content with tags.
    */
   tags?: string[];
+  /**
+   * A list of attributes for the collect NFT.
+   *
+   * This is the NFT description visible on marketplaces like OpenSea.
+   */
+  attributes?: MetadataAttribute[];
+  /**
+   * The collect NFT image.
+   *
+   * This is the NFT image visible on marketplaces like OpenSea.
+   *
+   * DO NOT use this as primary storage for publication media. Use the `media` property instead.
+   * In the case your publication has many media consider to use this field as a static representation
+   * of the collect NFT. For example if the publication is an album of audio files this could well be
```

Comment:
nit: 'well' is redundant here 
```suggestion
   * of the collect NFT. For example if the publication is an album of audio files this could be
```